dist_tables/ddl.rst

```diff
@@ -148,3 +148,70 @@ You can use the standard PostgreSQL DROP TABLE command to remove your distribute
 ::
 
     DROP TABLE github_events;
+
+.. _ddl_prop_support:
+
+Modifying Tables
+----------------
+
+Citus automatically propagates many kinds of DDL statements, which means that modifying a distributed table on the coordinator node will update shards on the workers too. Other DDL statements are unsupported on distributed tables, especially those which would modify a distribution column. Attempting to run unsupoorted DDL will raise an error and leave tables on the coordinator node unchanged. Some constraints like primary keys and uniqueness can only be applied prior to distributing a table.
+
+Here is a reference of the categories of DDL statements and whether the current version of Citus supports their propagation. (Note that automatic propagation can be enabled or disabled with a :ref:`configuration parameter <enable_ddl_prop>`.)
+
+General
+~~~~~~~
+
++------------+-----------------------------------+--------------------------------+
+| Propagates | DDL                               | Notes                          |
++============+===================================+================================+
+| YES        | Adding a Column                   |                                |
++------------+-----------------------------------+--------------------------------+
+| YES        | Removing a Column                 | Except the distribution column |
++------------+-----------------------------------+--------------------------------+
+| YES        | Changing a Column's Default Value |                                |
++------------+-----------------------------------+--------------------------------+
+| YES        | Changing a Column's Data Type     | Except the distribution column |
++------------+-----------------------------------+--------------------------------+
+| NO         | Renaming a Column                 |                                |
```

Comment:
Note that I tested each of these commands on Citus 6.1-17 so some of these YES/NO settings will not be up to date for 6.2. For instance renaming a column will be supported in 6.2 (see #285).

One option is to cherry pick this feature support matrix into 6.1 and add another commit on master to update the entries for 6.2. Or just list the correct data for 6.2 and don't include this section in 6.1.